stdlib/public/Cxx/CxxDictionary.swift

```diff
@@ -29,6 +29,9 @@ public protocol CxxDictionary<Key, Value> {
   func __findUnsafe(_ key: Key) -> RawIterator
 
   /// Do not implement this function manually in Swift.
+
```

Comment:
Let's keep the documentation comment for `__findMutatingUnsafe`, and let's have the same comment for `__beginUnsafe` as well.